x-pack/plugins/security_solution/public/management/hooks/use_with_show_responder.tsx

```diff
@@ -32,6 +32,7 @@ type ShowResponseActionsConsole = (props: ResponderInfoProps) => void;
 export interface BasicConsoleProps {
   agentId: string;
   hostName: string;
+  agentType: ResponseActionAgentType;
```

Comment:
do we need to add this here? it's also added below in `ResponserInfoProps`

Reply:
Yeah we do. If you notice the ones below are narrowed for specific `agentType`